spec/graphql/resolvers/add_on_resolver_spec.rb

```diff
@@ -29,6 +29,10 @@
   end
 
   it 'returns a single add-on' do
+    3.times do
```

Comment:
You should put this into a `before` block

Reply:
All done ! Thanks for your comments